index.bs

```diff
@@ -4386,7 +4428,7 @@ network.Cookie = {
     size: js-uint,
     httpOnly: bool,
     secure: bool,
-    sameSite: "strict" / "lax" / "none",
+    sameSite: network.SameSite,
```

Comment:
Probably, along with `domain` and `path, user wants to know `port` as well.

Reply:
Of cause, this can be done out of scope of this PR

Reply:
I believe `domain`, `path`, and the other properties listed here are cookie attributes:

<img width="653" alt="image" src="https://github.com/w3c/webdriver-bidi/assets/81942/6220f447-e2d8-46d9-aae6-8c19e2a25610">

`port` is not a cookie attribute.